sdk/dns/Azure.ResourceManager.Dns/tests/DnsManagementClientBase.cs

```diff
@@ -6,12 +6,13 @@
 using Azure.Core.TestFramework;
```

Comment:
Can we remove any Dns test specific things to focus just on the modifications to the test base classes that all RPs will use